shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/utils/LabelsBuilder.java

```diff
@@ -17,11 +17,14 @@
 
     public static final String OPERATOR_NAME = "bridge-fleet-shard-operator";
 
+    // Keep in sync with resources/templates kubernetes templates
+    public static final String BRIDGE_INGRESS_APPLICATION_TYPE = "bridge-ingress";
     /**
      * The tool being used to manage the operation of an application
      */
     public static final String MANAGED_BY_LABEL = "app.kubernetes.io/managed-by";
     public static final String CREATED_BY_LABEL = "app.kubernetes.io/created-by";
+    public static final String APPLICATION_TYPE_LABEL = "app.kubernetes.io/application-type";
```

Comment:
We are going to deploy multiple `Deployment` resources (i.e. `bridge-ingress` and `executor`), so we need to filter by `application-type` in the controllers. 
If you have any suggestion for the label name I'm all ears

Reply:
Why not use the existing `app.kubernetes.io/component`? 🤔

Reply:
:+1: done